kong/db/strategies/postgres/connector.lua

```diff
@@ -922,6 +932,11 @@ function _M.new(kong_config)
     cafile      = kong_config.lua_ssl_trusted_certificate_combined,
     sem_max     = kong_config.pg_max_concurrent_queries or 0,
     sem_timeout = (kong_config.pg_semaphore_timeout or 60000) / 1000,
+    pool_size   = kong_config.pg_pool_size or nil,
+    backlog     = kong_config.pg_backlog or nil,
```

Comment:
should be ok to remove `or nil`, will be `nil` by default
```suggestion
    pool_size   = kong_config.pg_pool_size,
    backlog     = kong_config.pg_backlog,
```

Reply:
Updated